### PR TITLE
Enable zuul pods for light checks

### PR DESCRIPTION
**Pull request by cjeanner**
Zuul pods are containers, allowing to get lightweight checks running
with less resources (like Prow CI).

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

File: roles/devscripts/tasks/111_verify_cluster.yml
Comment (cjeanner) [reply roles/devscripts/tasks/111_verify_cluster.yml:22]: with ansible-lint, it fails, because it apparently wants that one instead (probably because kubernetes.core.k8s_auth is mostly a wrapper around that one?)

File: roles/test_operator/tasks/main.yml
Comment (cjeanner) [reply roles/test_operator/tasks/main.yml:1]: it's needed, unless we add ignores to the rules to get the job in.... And I hate doing this....

File: roles/test_operator/tasks/main.yml
Comment (cjeanner) [reply roles/test_operator/tasks/main.yml:111]: I'll correct that one, right. And the next one as well.

**Comment by cjeanner** [comment]
/hold

need to fix the json_query things.

File: roles/devscripts/tasks/111_verify_cluster.yml
Comment (cjeanner) [reply roles/devscripts/tasks/111_verify_cluster.yml:22]: (note that the pod seems to apply different rules, or maybe uses another version - since we're not seeing this in the prow job o_O)

File: zuul.d/projects.yaml
Comment (cjeanner) [reply zuul.d/projects.yaml:5]: yep, that's the idea - but since that one is, for now, non-voting (to get some time to compare once it's merged).... :)

File: roles/devscripts/tasks/111_verify_cluster.yml
Comment (cjeanner) [reply roles/devscripts/tasks/111_verify_cluster.yml:22]: pushed a one-line patch to see if it triggers the ansible-lint issue. I really don't remember if it was local on my env, or indeed somewhere in zuul... "too old" for my brain.

**Comment by cjeanner** [comment]
/hold

Waiting next Monday to get it in

**Comment by cjeanner** [comment]
/retest

ansible-galaxy crashing with Timeout...

**Comment by cjeanner** [comment]
/hold cancel

We're Monday, let's get it in!

**Review by marios — APPROVED** [review]
lgtm

**Pull request merged**
